### PR TITLE
Fix manifest path for run from container

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

File: openshift_cli_installer/libs/rosa_clusters.py [hunk 1]
```diff
@@ -178,15 +178,19 @@ def destroy_hypershift_vpc(cluster_data):
...
 
 def prepare_hypershift_vpc(cluster_data):
-    vpc_file_path = os.path.join(
+    manifests_path = os.path.join(
         find_spec("openshift_cli_installer").submodule_search_locations[0],
-        "manifests/setup-vpc.tf",
+        "manifests",
     )
+    if not os.path.exists(manifests_path):
+        manifests_path = "openshift_cli_installer/manifests"
```
Comment (rnetser) [inline openshift_cli_installer/libs/rosa_clusters.py:186]: should be extracted to function

File: openshift_cli_installer/libs/rosa_clusters.py [hunk 2]
```diff
@@ -178,15 +178,19 @@ def destroy_hypershift_vpc(cluster_data):
...
 
-    shutil.copy(vpc_file_path, cluster_data["install-dir"])
+    shutil.copy(
+        os.path.join(manifests_path, "setup-vpc.tf"), cluster_data["install-dir"]
+    )
     terraform = terraform_init(cluster_data=cluster_data)
     try:
-        terraform.plan(dir_or_plan="rosa.plan")
+        terraform.plan("rosa.plan")
```
Comment (rnetser) [inline openshift_cli_installer/libs/rosa_clusters.py:193]: ok to drop the arg name?

**Review by rnetser — CHANGES_REQUESTED** [review]

File: openshift_cli_installer/libs/rosa_clusters.py (hunk 2 above)
Comment (myakove) [reply openshift_cli_installer/libs/rosa_clusters.py:187]: added again

File: openshift_cli_installer/libs/rosa_clusters.py [hunk 3]
```diff
@@ -172,35 +172,31 @@ def destroy_hypershift_vpc(cluster_data):
...
         return cluster_data
     except TerraformCommandError:
-        # Clean up already created resources from the plan
-        terraform.destroy(
-            force=IsNotFlagged,
-            auto_approve=True,
-            capture_output=False,
-            raise_on_error=True,
+        click.secho(
+            f"Create hypershift PVC for cluster {cluster_data['name']} failed, rolling back."
```
Comment (rnetser) [inline openshift_cli_installer/libs/rosa_clusters.py:196]:
```suggestion
            f"Create hypershift VPC for cluster {cluster_data['name']} failed, rolling back."
```

**Review by rnetser — CHANGES_REQUESTED** [review]

File: openshift_cli_installer/libs/rosa_clusters.py [hunk 4]
```diff
@@ -168,39 +168,36 @@ def terraform_init(cluster_data):
 
 
 def destroy_hypershift_vpc(cluster_data):
+    click.echo(f"Destroy hypershift PVC for cluster {cluster_data['name']}")
```
Comment (rnetser) [inline openshift_cli_installer/libs/rosa_clusters.py:171]:
```suggestion
    click.echo(f"Destroy hypershift VPC for cluster {cluster_data['name']}")
```

File: openshift_cli_installer/libs/rosa_clusters.py [hunk 5]
```diff
@@ -168,39 +168,36 @@ def terraform_init(cluster_data):
...
-        find_spec("openshift_cli_installer").submodule_search_locations[0],
-        "manifests/setup-vpc.tf",
+    shutil.copy(
+        os.path.join(get_manifests_path(), "setup-vpc.tf"), cluster_data["install-dir"]
     )
-
-    shutil.copy(vpc_file_path, cluster_data["install-dir"])
     terraform = terraform_init(cluster_data=cluster_data)
     try:
+        click.echo(f"Preparing hypershift PVC for cluster {cluster_data['name']}")
```
Comment (rnetser) [inline openshift_cli_installer/libs/rosa_clusters.py:187]:
```suggestion
        click.echo(f"Preparing hypershift VPC for cluster {cluster_data['name']}")
```

**Review by rnetser — CHANGES_REQUESTED** [review]

File: openshift_cli_installer/libs/rosa_clusters.py [hunk 6]
```diff
@@ -168,39 +168,36 @@ def terraform_init(cluster_data):
...
-        terraform.destroy(
-            force=IsNotFlagged,
-            auto_approve=True,
-            capture_output=False,
-            raise_on_error=True,
+        click.secho(
+            f"Create hypershift VPC for cluster {cluster_data['name']} failed, rolling back."
         )
+        # Clean up already created resources from the plan
+        destroy_hypershift_vpc(cluster_data=cluster_data)
```
Comment (rnetser) [inline openshift_cli_installer/libs/rosa_clusters.py:200]: should also call here `delete_oidc`

**Review by rnetser — APPROVED** [review]

**Pull request merged**